.github/workflows/validate.yml

```
@@ -12,25 +12,35 @@ jobs:
       CI: true
     steps:
       - name: Check out repo
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: pnpm/action-setup@v2
 
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
-          cache: yarn
+          cache: pnpm
           node-version-file: package.json
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: pnpm install --frozen-lockfile
+
+      - name: Cache Puppeteer
+        uses: actions/cache@v3
+        with:
+          path: .cache
```

This might contain too much stuff we don't want

```suggestion
					# As per https://pptr.dev/#installation
          path: $HOME/.cache/puppeteer
```

---

Maybe? AFAIK nothing else is making a `.cache` folder. It would be nice not to have to change the default cache location though.